GnuPlot.cs

```diff
@@ -423,7 +424,11 @@ public static void WriteData(double[] x, double[] y, double[] z, StreamWriter st
             {
                 if (i > 0 && x[i] != x[i - 1])
                     stream.WriteLine("");
-                stream.WriteLine(x[i] + " " + y[i] + " " + z[i]);
+
```

Comment:
That's nifty! Didn't like that line at all but didn't think about any other way to do it. Thanks! Fixed.